src/main/resources/assets/packagedauto/lang/ja_jp.json

```diff
@@ -0,0 +1,74 @@
+{
+	"itemGroup.packagedauto": "PackagedAuto",
+
+	"block.packagedauto.encoder": "パッケージレシピエンコーダー",
+	"block.packagedauto.encoder.save": "保存",
+	"block.packagedauto.encoder.save_single": "一つ保存",
+	"block.packagedauto.encoder.load": "読み込み",
+	"block.packagedauto.encoder.load_single": "一つ読み込み",
+	"block.packagedauto.encoder.pattern_slot": "レシピスロット %s",
+	"block.packagedauto.encoder.change_recipe_type": "レシピタイプ変更",
+	"block.packagedauto.packager": "パッケージャー",
+	"block.packagedauto.packager.mode.exact": "モード:正確性を要求",
+	"block.packagedauto.packager.mode.disjoint": "モード:不一致を要求",
+	"block.packagedauto.packager.mode.first": "モード:最初のマッチ",
+	"block.packagedauto.packager_extension": "パッケージャー拡張",
+	"block.packagedauto.unpackager": "アンパッケージャー",
+	"block.packagedauto.unpackager.blocking.false": "ブロッキングモードが無効",
+	"block.packagedauto.unpackager.blocking.true": "ブロッキングモードが有効",
+	"block.packagedauto.unpackager.tracker.decrease": "トラッカー数を減らす",
+	"block.packagedauto.unpackager.tracker.increase": "トラッカー数を増やす",
+	"block.packagedauto.distributor": "ポジションパッケージディストリビューター",
+	"block.packagedauto.crafter": "パッケージクラフター",
+	"block.packagedauto.fluid_package_filler": "液体パッケージフィラー",
+	"block.packagedauto.fluid_package_filler.redstone": "レッドストーンで有効化して開始します",
+	"block.packagedauto.packaging_provider": "MEパッケージプロバイダー",
+	"block.packagedauto.packaging_provider.direct.false": "直接レシピを提供しない",
+	"block.packagedauto.packaging_provider.direct.true": "直接レシピを提供する",
+	"block.packagedauto.packaging_provider.packaging.false": "パッケージレシピを提供しない",
+	"block.packagedauto.packaging_provider.packaging.true": "パッケージレシピを提供する",
+	"block.packagedauto.packaging_provider.unpackaging.false": "アンパッケージレシピを提供しない",
+	"block.packagedauto.packaging_provider.unpackaging.true": "アンパッケージレシピを提供する",
+
+	"item.packagedauto.recipe_holder": "パッケージレシピホルダー",
+	"item.packagedauto.recipe_holder.recipes": "レシピ:",
+	"item.packagedauto.distributor_marker": "ディストリビューターマーカー",
+	"item.packagedauto.distributor_marker.dimension": "ディメンション: %s",
+	"item.packagedauto.distributor_marker.position": "ポジション: %s",
+	"item.packagedauto.distributor_marker.direction": "方向: %s",
+	"item.packagedauto.package": "レシピパッケージ",
+	"item.packagedauto.package.index": "索引 : %d",
+	"item.packagedauto.package.items": "アイテム:",
+	"item.packagedauto.volume_package": "ボリュームパッケージ",
```

Comment:
The translation of "Volume" here should probably be consistent with the ui title